Utils/utils/action_get_current_time.py

```diff
@@ -49,10 +57,28 @@ def _utc_to_gmt(self, value):
         return result_time
 
     def run(self, ra: Arguments) -> dict:
-        self.log(message=f"Retrieving current time for {ra.selectedTimezone}", level="info")
-        dateToReturn = self._utc_to_gmt(ra.selectedTimezone)
+        if not ra.selectedNamedTimezone and not ra.selectedTimezone:
+            self.log(message="You should set a timezone", level="error")
+            return {}
+
+        # new field has a higher priority
+        if ra.selectedNamedTimezone:
+            self.log(message=f"Retrieving current time for {ra.selectedNamedTimezone}", level="info")
+            try:
+                tz = ZoneInfo(ra.selectedNamedTimezone)
+
+            except ZoneInfoNotFoundError as err:
+                self.log_exception(err)
+                return {}
```

Comment:
Can you raise an exception? In order to stop the workflow after this action
```suggestion
            raise Exception(f"Invalid timezone: {ra.selectedNamedTimezone}")
```